crates/ibc/src/core/ics24_host/identifier.rs

```diff
@@ -105,6 +120,21 @@ impl ChainId {
         self.revision_number = revision_number;
     }
 
+    /// Unsets the revision number of the `ChainId`
+    /// ```
+    /// use ibc::core::ics24_host::identifier::ChainId;
+    /// let mut chain_id = ChainId::new("chainA", 1).unwrap();
+    /// chain_id.unset_revision_number();
+    /// assert!(!chain_id.has_revision_number());
+    /// assert_eq!(chain_id.revision_number(), 0);
+    /// assert_eq!(chain_id.as_str(), "chainA");
+    /// ```
+    pub fn unset_revision_number(&mut self) {
+        let chain_name = self.chain_name();
+        self.id = chain_name.to_string();
+        self.revision_number = 0;
+    }
```

Comment:
Since we have maintained the `ChainId` data structure in its original form, there is no longer a need for this method. The `ChainId` whether adheres to the specified format, where the concept of `unset` no longer applies or the `revision_number` consistently remains at zero for all the rest cases. right?